### DISCUSSION TITLE
Error faillure to get Novel

### DISCUSSION THREAD

**Discussion by tahirieljalil** [lncrawl/lightnovel-crawler#2357] [Q&A]
Hi guys i think the last update doens't working look at the image attachet 
![Capture d'écran 2024-04-23 092500](https://github.com/dipu-bd/lightnovel-crawler/assets/4559137/1b1970ea-9d70-422f-957b-57bd684e8a68)

**Comment by lukholo3015** [comment]
I have the same issue!
Looking for alternatives

**Comment by lukholo3015** [comment]
FOUND SOLUTION!
Use url method (NOTE: Some sources still don't work)
                      [#] Lightnovel Crawler v3.5.0
              https://github.com/dipu-bd/lightnovel-crawler
-> Press Ctrl + C to exit

? Enter novel page url or query novel: https://boxnovel.com/novel/the-eternal-supreme/
Retrieving novel info...

[#] The Eternal Supreme
23 volumes and 2210 chapters found.

https://boxnovel.com/novel/the-eternal-supreme/
? Enter output directory: C:\Users\Admin\Downloads\Lightnovels\boxnovel-com\The Eternal Supreme
? Which chapters to download? First 10 chapters
? 10 chapters selected Continue
? Which output formats to create? done (4 selections)
? How many files to generate? Pack everything into a single file
Chapters: 100%|██████████████████████████████████████████████████████████████████████| 10/10 [00:03<00:00, 3.19item/s]
Images: 100%|████████████████████████████████████████████████████████████████████████| 1/1 [00:00<00:00, 3.84item/s]
Created: The Eternal Supreme c1-10.epub
Created: 10 text files

Task completed
? Open the output folder? Yes
Press ENTER to exit...

**Reply by lukholo3015** [reply]
![Screenshot 2024-04-28 122202](https://github.com/dipu-bd/lightnovel-crawler/assets/44092175/c014d3bb-66c5-453c-a4be-90d1bfdfb7c8)

**Comment by tahirieljalil** [comment]
Thank you, it works! You are a lifesaver 
![image](https://github.com/dipu-bd/lightnovel-crawler/assets/4559137/6a9e01aa-4c81-4d81-a361-ded35f16ecbd)
